content/docs/reference/pkg/alicloud/adb/getclusters.md

```diff
@@ -366,11 +358,19 @@ The following output properties are available:
     <dd>{{% md %}}A list of ADB cluster IDs. 
 {{% /md %}}</dd>
 
+    <dt class="property-"
```

Comment:
Not this PR I assume,but this doesn't look right; presumably something amiss here: https://github.com/pulumi/pulumi/blob/729ae08c9d4bf0099e6382ac5d176540ac16f98a/pkg/codegen/docs/templates/properties.tmpl#L7